darts/tests/models/forecasting/test_residuals.py

```diff
@@ -612,3 +619,304 @@ def test_sample_weight(self, config):
         for res_nw, res_w in zip(res_non_weighted, res_weighted):
             with pytest.raises(AssertionError):
                 np.testing.assert_array_almost_equal(res_w, res_nw)
+
+    @pytest.mark.parametrize(
+        "config",
+        itertools.product(
+            [True, False],  # last_points_only
+            [False, True],  # from stochastic predictions (or predicted quantiles)
+        ),
+    )
+    def test_quantiles_metrics(self, config):
+        """Tests backtest with different metric_kwargs based on historical forecasts generated on a sequence
+        `series` with last_points_only=False"""
```

Comment:
The comment is contradictory with the parameters in the product, where lpo can take either the `True` or `False` value.